backend/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Python base image
+FROM python:3.10-slim
```

Comment:
- I forgot to change the image name as my purpose was to test this FastAPI running with docker compose. I will switch to python version 3.11.10.
- The slim family is similar to alpine but sometimes it can be larger on size. We can use the alpine family.